Index: b/insserv.c
===================================================================
--- a/insserv.c
+++ b/insserv.c
@@ -998,7 +998,7 @@ char *myname = (char*)0;
 static void _logger (const char *restrict const fmt, va_list ap);
 static void _logger (const char *restrict const fmt, va_list ap)
 {
-    extension char buf[strlen(myname)+2+strlen(fmt)+1];
+    __extension__ char buf[strlen(myname)+2+strlen(fmt)+1];
     strcat(strcat(strcpy(buf, myname), ": "), fmt);
     vfprintf(stderr, buf, ap);
     return;
@@ -2382,7 +2382,7 @@ int main (int argc, char *argv[])
     DIR * initdir;
     struct dirent *d;
     struct stat st_script;
-    extension char * argr[argc];
+    __extension__ char * argr[argc];
     char * path = INITDIR;
     char * override_path = OVERRIDEDIR;
     char * insconf = INSCONF;
